waterloop-site/src/components/PastGeeseTimeline/ImageCarousel.tsx

```diff
@@ -1,13 +1,14 @@
 import React from "react";
 import { useSwipeable } from "react-swipeable";
 import styled from "styled-components";
-import useGeeseImages from "./hooks/geese-images";
+import { useGeeseImages, imgs } from "./hooks/geese-images";
```

Comment:
imgs should be returned from useGeeseImages, not exported from this file. Eventually, it will have to be something that comes from an API call, so it will need to be fetched inside the hook when that time comes

Reply:
Gotcha, this has been changed.